plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -145,6 +145,127 @@ func NoDiskConflict(pod *api.Pod, existingPods []*api.Pod, node string) (bool, e
 	return true, nil
 }
 
+type MaxEBSVolumeCountChecker struct {
```

Comment:
Added on the `New` function like the other scheduler predicate factory functions.